Sources/StoryKit/Stories/Common/Models/Story.swift

```diff
@@ -78,3 +78,9 @@ public extension StoryKit.Story {
         }
     }
 }
+
+public extension StoryKit.Story.PageData {
+    var buttonLink: URL? {
+        buttonData?.link
+    }
+}
```

Comment:
Why did you add extension instead of just make public  `link` property in `ButtonData` so you will be able to get link by` .buttonData?.link`?